runtime/ftplugin/help.lua

```diff
@@ -1,7 +1,13 @@
 -- use treesitter over syntax (for highlighted code blocks)
 vim.treesitter.start()
 
--- add custom highlights for list in `:h highlight-groups`
+-- Higlight "===" and "---" heading delimiters specially.
+vim.cmd[[
+  hi @markup.heading.delimiter1.vimdoc guibg=bg guifg=bg guisp=fg gui=underdouble
```

Comment:
Any reason you're not simply using `@markup.heading.delimiter.1.vimdoc`, which would be the documented approach instead of a custom "hack"?

Reply:
(I used to have the `.delimiter` groups but removed them due to pressure. If we really want to make use of them, we should officially reintroduce and document them (and use them also in markdown etc.; would be a lot of extra work for me in nvim-treesitter, though...) 

As they are part of the fallback hierarchy, they would work gracefully.